app/controllers/api/v1/houses_controller.rb

```diff
@@ -0,0 +1,154 @@
+class Api::V1::HousesController < ApplicationController
+  # load_and_authorize_resource
+  before_action :set_house, only: %i[show edit update destroy]
+
+  # GET /houses or /houses.json
+  def index
+    @houses = House.all
+    render json: @houses
+  end
+
+  # GET /houses/1 or /houses/1.json
+  def show; end
+
+  # GET /houses/new
+  def new
+    @house = House.new
+  end
+
+  # GET /houses/1/edit
+  def edit; end
+
+  # POST /houses or /houses.json
+  def create
+    @house = House.new(house_params)
+
+    if @house.save
+      render json: { status: 'success', message: 'House created successfully!' }, status: :created
+    else
+      render json: { status: 'failed', errors: @house.errors.full_messages }, status: :unprocessable_entity
+    end
+  end
+
+  # PATCH/PUT /houses/1 or /houses/1.json
+  def update
+    respond_to do |format|
+      if @house.update(house_params)
+        format.html { redirect_to house_url(@house), notice: 'House was successfully updated.' }
+        format.json { render :show, status: :ok, location: @house }
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+        format.json { render json: @house.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # DELETE /houses/1 or /houses/1.json
+  def destroy
+    @house.destroy
+
+    respond_to do |format|
+      format.html { redirect_to houses_url, notice: 'House was successfully destroyed.' }
+      format.json { head :no_content }
+    end
+  end
```

Comment:
- [ ] What happens if the house is not successfully destroyed?  Kindly, add some error handling in this action. 👍